src/content/getting-started/designers/designers.md

```diff
@@ -2,16 +2,38 @@
 title: Designers
 ---
 
-This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Carbon Design Kit"
```

Comment:
Change to "Download from GitHub" and change icon to GitHub icon

Reply:
If we don't change the title of the section then we need to say "Download the kit from GitHub"